### PR TITLE
Thu/parse date as utc prod 11811

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/DateUtils/DateUtils.js [hunk 1]
```diff
@@ -1,29 +1,84 @@
...
+const decreaseDateByOffset = (date, offset) => {
+  if (date instanceof Date === false) {
+    console.error('decreaseDateByOffset parameter "date" must be a Date object');
+    return null;
+  }
+  return new Date(date - offset * 60 * 1000);
+};
+
+const increaseDateByOffset = (date, offset) => {
+  if (date instanceof Date === false) {
```
Comment (nborde-CSM) [inline src/DateUtils/DateUtils.js:21]: As below you call the `decreaseDateByOffset` function that already check if _date_ is a Date object, is it really interesting to check it second time in this function ?

File: src/DateUtils/__test__/DateUtils_timezones.spec.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    ${undefined}
+    ${''}
+    ${'foo'}
+    ${0}
+    ${1}
+    ${{}}
+    ${[]}
+    ${'06/32/2022'}
+  `('should return null if input is "$parameter"', ({ parameter }) => {
+    const mockError = jest.spyOn(console, 'error').mockImplementation(() => {});
```
Comment (nborde-CSM) [inline src/DateUtils/__test__/DateUtils_timezones.spec.js:97]: This mock is used several time. Shouldn't it define outside describe block ?

**Review by nborde-CSM — APPROVED** [review]
Some little remarks but LGTM :+1:

File: src/DateUtils/__test__/DateUtils_timezones.spec.js (hunk 2 above)
Comment (csm-thu) [reply src/DateUtils/__test__/DateUtils_timezones.spec.js]: Fixed it :+1: 
Thank you for the feedback

**Pull request merged**